### PR TITLE
Ref Guide - CVE Audit

### PR DESCRIPTION
**Pull request by Loquacity**
https://github.com/SUSE/spacewalk/issues/10588

Moves content to new chapter in Admin Guide, then references it.

New chapter will require editing and updated screenshots, (will do in new PR). It's here: https://github.com/uyuni-project/uyuni-docs/pull/87

### CODE REVIEW

File: modules/administration/pages/auditing.adoc [hunk 1]
```diff
@@ -0,0 +1,108 @@
+[[ref.webui.audit.cve]]
```
Comment (keichwa) [inline modules/administration/pages/auditing.adoc]: This needs editing.  Maybe, I can work on it later today.

File: modules/reference/pages/audit/audit-cve-audit.adoc [hunk 2]
```diff
@@ -1,108 +1,25 @@
 [[ref.webui.audit.cve]]
 = CVE Audit
 
-The menu:Main Menu[Audit > CVE Audit] page will display a list of client systems with their patch status regarding a given CVE (Common Vulnerabilities and Exposures) number.
+The menu:Audit[CVE Audit] section shows you which CVEs have been applied to your clients.
+A CVE (common vulnerabilities and exposures) is a fix for a publicly known cybersecurity vulnerability.
```
Comment (keichwa) [inline modules/reference/pages/audit/audit-cve-audit.adoc]:
```suggestion
A CVE (common vulnerabilities and exposures) is a fix for a publicly known security vulnerability.
```

File: modules/reference/pages/audit/audit-cve-audit.adoc [hunk 3]
```diff
@@ -1,108 +1,25 @@
...
+It is extremely important that you apply CVEs to your clients as soon as they become available.
```
Comment (keichwa) [inline modules/reference/pages/audit/audit-cve-audit.adoc]:
```suggestion
It is important that you apply CVEs to your clients as soon as they become available.
```
Or we can use a formal admon block.

Comment (Loquacity) [reply modules/reference/pages/audit/audit-cve-audit.adoc]: yeah, this could warrant an [IMPORTANT]

**Pull request merged**